capstone/src/main/webapp/WEB-INF/datastore-indexes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This one should also not be necessary :) (unless this did help and I'm missing something?)